src/Utils.js

```diff
@@ -213,6 +213,19 @@ export function coercePropValue(propName, propValue) {
     return propValue;
   }
 
+  // can be the empty string
+  if (propValue === '') {
+    return propValue;
+  }
+
+  if (is('NaN', propValue)) {
```

Comment:
i'll change it; i'd been using a literal `NaN` and then updated it to take in a string